bibliography/ch9_fault_attacks.bib

```diff
@@ -0,0 +1,63 @@
+@CONFERENCE {bib:nohl2008,
```

Comment:
Need to add these into the existing english.bib file.

Reply:
Done